<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wsfeForm</class>
 <widget class="QWidget" name="wsfeForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Url a conectar:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="hostNameEdit"/>
     </item>
     <item>
      <widget class="QPushButton" name="connectButton">
       <property name="text">
        <string>Conectar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Operación:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="operationComboBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>Obtener CAE p/Comprobante</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Obtener Info de Comprobante</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Obtener Ult. Comprobante</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Informacion</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="operationHelperComboBox">
       <item>
        <property name="text">
         <string>FEParamGetTiposCbte</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetTiposConcepto</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetTiposDoc</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetTiposIva</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetTiposMonedas</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetTiposOpcional</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetTiposTributos</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetPtosVenta</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEParamGetCotizacion</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>FEDummy</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="doButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Ejecutar Operación</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Operaciones</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="puntoVentaLabel">
            <property name="text">
             <string>Punto Venta:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="puntoVentaSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="tipoComprobanteLabel">
            <property name="text">
             <string>Tipo Comprobante:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="tipoComprobanteSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>99999999</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="nroComprobanteLabel">
            <property name="text">
             <string>Nro. Comprobante:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="nroComprobanteSpinBox">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="buttonSymbols">
             <enum>QAbstractSpinBox::NoButtons</enum>
            </property>
            <property name="maximum">
             <number>999999</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="sessionOutput"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
